tailscale/DOCS.md

```diff
@@ -17,8 +17,55 @@ the following URL:
 
 <https://login.tailscale.com/start>
 
-You can also create an account during the add-on installation processes,
-however, it is nice to know where you need to go later on.
+<details>
+<summary>
```

Comment:
Let's not do those weird collapsable, this is not a 10 pages doc.
The fact you have to write `(click to open)` is already a good indication it is bad ux.